### PR TITLE
Test: accelerate integrate tests

### LINKED ISSUE #4357: Running integration tests is too time-consuming
### Describe the Testing Issue

Running integration tests is too time-consuming. I plan to update the parameters to accelerate integration tests under the premise that the test function remains unchanged.

### Additional Context

_No response_

### Task list for Issue attackers (only for developers)

- [ ] Understand the testing issue described by the developer.
- [ ] Review the specific test case, expected and actual results, and any error messages.
- [ ] Identify the root cause of the test failure or issue.
- [ ] If a possible solution is suggested, evaluate its feasibility and effectiveness.
- [ ] Implement a fix for the test failure or issue, or create a new test case if needed.
- [ ] Verify that the fix resolves the testing issue and the test case passes.
- [ ] Review and update any relevant documentation, such as test plans or user guides.
- [ ] Ensure the testing issue is resolved and close the ticket.
- [ ] Share any lessons learned or best practices with the team to prevent similar issues in the future.

### PR DESCRIPTION
**Pull request by YuLiu98**
### Linked Issue
Fix #4357

### CODE REVIEW

**Review by mohanchen — APPROVED** [review]

File: tests/integrate/204_NO_KP_AFM/INPUT [hunk 1]
```diff
@@ -8,7 +8,7 @@ orbital_dir	../../PP_ORB
 gamma_only      0
 
 #Parameters (2.Iteration)
-ecutwfc			50
+ecutwfc			20
```
Comment (dyzheng) [inline tests/integrate/204_NO_KP_AFM/INPUT:11]: Please be care of the convergent magnetizations of 204*/304* should be non-zero.

Comment (YuLiu98) [reply tests/integrate/204_NO_KP_AFM/INPUT:11]: > Please be care of the convergent magnetizations of 204*/304* should be non-zero.

I see. However, it seems that the original setting cannot get non-zero magnetic moments. Anyway, I have solved the problem.

**Review by dyzheng — APPROVED** [review]

**Review by WHUweiqingzhou — CHANGES_REQUESTED** [review]
I find some warnings in CI tests for the cases you update the ref data in this PR, which confuses me. f.e:
103_PW_15_CF_CS_S1_smallg
103_PW_15_CF_CS_S2_smallg
601_NO_TDDFT_CO
601_NO_TDDFT_CO_occ

I do not list all of them here. How do you get the ref data?
According to https://github.com/deepmodeling/abacus-develop/blob/develop/.github/workflows/test.yml and https://github.com/deepmodeling/abacus-develop/blob/develop/tests/integrate/CMakeLists.txt,
I believe CI test uses `OMP_NUM_THREADS=2 mpirun -np 4 abacus |tee log` to get result.

**Comment by YuLiu98** [comment]
> I find some warnings in CI tests for the cases you update the ref data in this PR, which confuses me. f.e: 103_PW_15_CF_CS_S1_smallg 103_PW_15_CF_CS_S2_smallg 601_NO_TDDFT_CO 601_NO_TDDFT_CO_occ
> 
> I do not list all of them here. How do you get the ref data? According to https://github.com/deepmodeling/abacus-develop/blob/develop/.github/workflows/test.yml and https://github.com/deepmodeling/abacus-develop/blob/develop/tests/integrate/CMakeLists.txt, I believe CI test uses `OMP_NUM_THREADS=2 mpirun -np 4 abacus |tee log` to get result.

Note that the original CI test reports 65 warnings, while this PR's CI test reports 48 warnings. I have reduced some warnings except for 601_NO_TDDFT_CO_occ. As I know from @lyb9812, this TDDFT example is unstable.

**Comment by WHUweiqingzhou** [comment]
> > I find some warnings in CI tests for the cases you update the ref data in this PR, which confuses me. f.e: 103_PW_15_CF_CS_S1_smallg 103_PW_15_CF_CS_S2_smallg 601_NO_TDDFT_CO 601_NO_TDDFT_CO_occ
> > I do not list all of them here. How do you get the ref data? According to https://github.com/deepmodeling/abacus-develop/blob/develop/.github/workflows/test.yml and https://github.com/deepmodeling/abacus-develop/blob/develop/tests/integrate/CMakeLists.txt, I believe CI test uses `OMP_NUM_THREADS=2 mpirun -np 4 abacus |tee log` to get result.
> 
> Note that the original CI test reports 65 warnings, while this PR's CI test reports 48 warnings. I have reduced some warnings except for 601_NO_TDDFT_CO_occ. As I know from @lyb9812, this TDDFT example is unstable.

how about these cases?
103_PW_15_CF_CS_S1_smallg
103_PW_15_CF_CS_S2_smallg
601_NO_TDDFT_CO

**Comment by YuLiu98** [comment]
> > > I find some warnings in CI tests for the cases you update the ref data in this PR, which confuses me. f.e: 103_PW_15_CF_CS_S1_smallg 103_PW_15_CF_CS_S2_smallg 601_NO_TDDFT_CO 601_NO_TDDFT_CO_occ
> > > I do not list all of them here. How do you get the ref data? According to https://github.com/deepmodeling/abacus-develop/blob/develop/.github/workflows/test.yml and https://github.com/deepmodeling/abacus-develop/blob/develop/tests/integrate/CMakeLists.txt, I believe CI test uses `OMP_NUM_THREADS=2 mpirun -np 4 abacus |tee log` to get result.
> > 
> > 
> > Note that the original CI test reports 65 warnings, while this PR's CI test reports 48 warnings. I have reduced some warnings except for 601_NO_TDDFT_CO_occ. As I know from @lyb9812, this TDDFT example is unstable.
> 
> how about these cases? 103_PW_15_CF_CS_S1_smallg 103_PW_15_CF_CS_S2_smallg 601_NO_TDDFT_CO

As I mentioned above, these tests already reported warnings before this PR. Maybe previous PRs should be checked to determine when these warnings are introduced.  

103_PW_15_CF_CS_S1_smallg  and 103_PW_15_CF_CS_S1_smallg do not report warnings after this PR.

**Review by WHUweiqingzhou — APPROVED** [review]
LGTM.

**Pull request merged**